<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PredictionWidget</class>
 <widget class="QWidget" name="PredictionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pr√©diction de Contenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #BDC3C7;
}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane {
    border: 2px solid #BDC3C7;
    border-radius: 15px;
    background-color: #BDC3C7;
}

QTabBar::tab {
    background: transparent;
    color: black;
    padding: 8px 20px;
    margin: 4px;
    border: 1px solid #999;
    border-radius: 15px;
    font-weight: bold;
}

QTabBar::tab:selected {
    background: #95A5A6;
    color: white;
}

QTabBar::tab:hover {
    background: #A7B1B1;
    color: white;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Pr√©diction</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>400</width>
       <height>400</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 2px solid #9B59B6;
    border-radius: 15px;
    margin-top: 20px;
    background: qlineargradient(spread:pad,
                                x1:0, y1:0,
                                x2:1, y2:0,
                                stop:0 #9B59B6,
                                stop:1 #BDC3C7);
    color: white;
    font-weight: bold;
    padding: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top centre;
    padding: 10px 20px;
    border-radius: 15px;
    background: qlineargradient(spread:pad,
                                x1:0, y1:0,
                                x2:1, y2:0,
                                stop:0 #9B59B6,
                                stop:1 #BDC3C7);
    color: black;
    font-size: 18px;
    font-weight: bold;
}</string>
     </property>
     <property name="title">
      <string>Informations sur le contenu</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent; color: black; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Type de contenu:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboTypeContenu">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>350</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white; color: black;</string>
      </property>
      <item>
       <property name="text">
        <string>Video courte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Video longue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Image</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Texte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Article</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Podcast</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>115</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent; color: black; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Public cible:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboPublicCible">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>350</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white; color: black;</string>
      </property>
      <item>
       <property name="text">
        <string>Jeunes (18-25 ans)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jeunes adultes (26-35 ans)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Professionnels</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Entreprises</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grand public</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>√âtudiants</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent; color: black; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Date de d√©but (optionnel):</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>205</y>
        <width>350</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white; color: black;</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>245</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent; color: black; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Dur√©e (secondes, pour vid√©os):</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinDuree">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>350</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white; color: black;</string>
      </property>
      <property name="maximum">
       <number>7200</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPredire">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>350</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    border-radius: 20px;
    color: white;
    padding: 10px;
    background: qlineargradient(spread:pad,
                                x1:0, y1:0,
                                x2:1, y2:0,
                                stop:0 #9B59B6,
                                stop:1 #BDC3C7);
    font-weight: bold;
    font-size: 14px;
}

QPushButton:hover {
    background: qlineargradient(spread:pad,
                                x1:0, y1:0,
                                x2:1, y2:0,
                                stop:0 #BDC3C7,
                                stop:1 #F8C8DC);
    color: black;
}</string>
      </property>
      <property name="text">
       <string>üîÆ Pr√©dire le meilleur moment et plateforme</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>520</width>
       <height>400</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 2px solid #F8C8DC;
    border-radius: 15px;
    margin-top: 20px;
    background: qlineargradient(spread:pad,
                                x1:0, y1:0,
                                x2:1, y2:0,
                                stop:0 #F8C8DC,
                                stop:1 #BDC3C7);
    color: white;
    font-weight: bold;
    padding: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top centre;
    padding: 10px 20px;
    border-radius: 15px;
    background: qlineargradient(spread:pad,
                                x1:0, y1:0,
                                x2:1, y2:0,
                                stop:0 #F8C8DC,
                                stop:1 #BDC3C7);
    color: black;
    font-size: 18px;
    font-weight: bold;
}</string>
     </property>
     <property name="title">
      <string>R√©sultats de la pr√©diction</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent; color: black; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>üìÖ Meilleur moment:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textMoment">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>480</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white; color: black; border-radius: 10px; padding: 10px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>135</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent; color: black; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>üåê Meilleure plateforme:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textPlateforme">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>480</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white; color: black; border-radius: 10px; padding: 10px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent; color: black; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>üí° Justification:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textJustification">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>245</y>
        <width>480</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white; color: black; border-radius: 10px; padding: 10px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
